lottie-ios/Classes/Extensions/LOTBezierPath.m

```diff
@@ -340,7 +340,6 @@ - (void)trimPathFromT:(CGFloat)fromT toT:(CGFloat)toT offset:(CGFloat)offset {
           toPoint = LOT_PointInLine(currentPoint, subpath->endPoint, currentSpanEndT);
         }
         [self LOT_addLineToPoint:toPoint];
-        currentPoint = toPoint;
```

Comment:
This line is still needed for the next iteration of the loop?

Reply:
@hansemannn Bump

Reply:
Addressed, thx!